org.sugarlabs.Falabracman.json

```diff
@@ -0,0 +1,92 @@
+{
+    "app-id": "org.sugarlabs.Falabracman",
+    "base": "org.sugarlabs.BaseApp",
+    "base-version": "24.04",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "separate-locales": false,
+    "command": "sugarapp",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--device=dri",
+        "--env=SUGAR_BUNDLE_ID=org.sugarlabs.Falabracman",
+        "--env=SUGAR_BUNDLE_PATH=/app/share/sugar/activities/Falabracman.activity"
+    ],
+    "modules": [
+        {
+            "name": "pygame2",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://files.pythonhosted.org/packages/c6/aa/2c0c867d6cff00966cfc2152b25f61599f87e88b239e4dcb8ad5357f0f69/pygame-2.5.2.tar.gz",
+                    "sha256": "c1b89eb5d539e7ac5cf75513125fb5f2f0a2d918b1fd6e981f23bf0ac1b1c24a",
+                    "x-checker-data": {
+                        "type": "pypi",
+                        "name": "pygame"
+                    }
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --ignore-installed --no-deps --prefix=/app ."
+            ],
+            "build-options": {
+                "env": {
+                    "PORTMIDI_INC_PORTTIME": "1",
+                    "PYGAME_EXTRA_BASE": "/app"
+                },
+                "arch": {
+                    "aarch64": {
+                        "env": {
+                            "ORIGLIBDIRS": "/lib:/lib64:/lib/aarch64-linux-gnu"
```

Comment:
Are these correct? Shouldn't it be `/app/lib` etc.

Reply:
If the application or build process needs to access system-wide libraries than the above configuration is good.

our [Pippy](https://github.com/flathub/org.sugarlabs.Pippy/blob/master/org.sugarlabs.Pippy.json) and [Physics](https://github.com/flathub/org.sugarlabs.Physics/blob/master/org.sugarlabs.Physics.json) are also using the same configurations